### ISSUE TITLE
Need enhancement where Sentry alerts can be added as a task to an existing Epic. 

### ISSUE THREAD

**Issue by uvkishore** [getsentry/sentry#55781]
### Problem Statement

Hi Team, 

we have an epic, and that epic will be a place holder for all the alerts and tickets that are created from Sentry, we want an enhancement that the new ticket that is created to Jira should be added as a task to an existing epic ticket.  This is we are not able to do it at the moment. 

### Product Area

Alerts

**Comment by ceorourke** [comment]
Hi @uvkishore without seeing your Jira setup it's hard to say, but looking at my own I have an "Epic" dropdown and an "Issue Type" dropdown - if you make the issue type "task" and select the right epic, does that not do what you want? Do you not have those options in your modal?

**Comment by uvkishore** [comment]
Hi Team ,

That is correct if we try to provide the epic , nothing happens , even the sentry would not create a ticket to Jira.




Kishore K

DevOps Consultant

***@***.******@***.***>

[cid:ultraviolet_logo_reverse_outline_rgb-250_29c82340-bac0-4721-a40b-3c2cfe6031eb.png]<https://www.uvcyber.com/>


UVCyber.com<https://www.uvcyber.com/>
Connect with us:
[cid:social_icon_linkedin-black-white-30_726d4a26-8e6b-45a2-a8a2-1a542716c9d3.png]<https://www.linkedin.com/company/uvcyber> [cid:social_icon_twitter-black-white-30_f3efe673-9c33-4bad-98eb-c56400e0a8bb.png] <https://twitter.com/uv_cyber>  [cid:social_icon_facebook-black-white-30_a7d8eae9-5950-408f-b3b0-82270170b787.png] <https://www.facebook.com/uvcyber/>

[cid:partner-logo-bar_ac92a414-f917-4eb1-9785-579b8af318af.png]

**Comment by ceorourke** [comment]
@uvkishore can you expand on that? Are you able to select an epic? If so, when you select it and click "create issue" what happens? Or are you perhaps referring to automatically creating a Jira ticket from an issue alert?

**Comment by uvkishore** [comment]
Yes, we can select Task and then provide the Epic number ( ASDR-2286) in the drop down . Hoping to create a task below that epic.

Once we do this and apply the changes , we will not get any tickets created to Jira. Please take a look at the below screenshot.

***@***.***




Kishore K

DevOps Consultant

***@***.******@***.***>

[cid:ultraviolet_logo_reverse_outline_rgb-250_29c82340-bac0-4721-a40b-3c2cfe6031eb.png]<https://www.uvcyber.com/>


UVCyber.com<https://www.uvcyber.com/>
Connect with us:
[cid:social_icon_linkedin-black-white-30_726d4a26-8e6b-45a2-a8a2-1a542716c9d3.png]<https://www.linkedin.com/company/uvcyber> [cid:social_icon_twitter-black-white-30_f3efe673-9c33-4bad-98eb-c56400e0a8bb.png] <https://twitter.com/uv_cyber>  [cid:social_icon_facebook-black-white-30_a7d8eae9-5950-408f-b3b0-82270170b787.png] <https://www.facebook.com/uvcyber/>

[cid:partner-logo-bar_ac92a414-f917-4eb1-9785-579b8af318af.png]

**Comment by uvkishore** [comment]
![image](https://github.com/getsentry/sentry/assets/130484874/34c32b39-72b1-4541-aea9-ee1ff6b0ef88)
